pegasus_wrapper/resource_request.py

```diff
@@ -125,6 +140,8 @@ def convert_time_to_slurm_format(self, job_time_in_minutes: int) -> str:
     def apply_to_job(self, job: Job, *, job_name: str) -> None:
         if not self.partition:
             raise RuntimeError("A partition to run on must be specified.")
+        exclude_list = self.exclude_list
+        run_on_single_node = self.run_on_single_node
```

Comment:
nit: I'm unsure why these need to be assigned to local variables before being used further down in the code?

Reply:
@hassanzadeh Is there a specific reason you were assigning these to local variables?